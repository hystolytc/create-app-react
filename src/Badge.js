import React from 'react'

const Badge = () => (
  <div>This is Badge component</div>
)

export default Badge