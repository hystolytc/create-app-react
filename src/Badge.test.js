import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render, screen } from '@testing-library/react'
import Badge from './Badge'

const describeComponent = '<Badge /> test case'

describe(describeComponent, () => {
  const { container } = render(<Badge />)

  test(`Render ${describeComponent}`, () => {
    expect(container).toMatchSnapshot()
  })
})