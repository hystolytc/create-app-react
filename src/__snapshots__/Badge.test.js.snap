// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> test case Render <Badge /> test case 1`] = `
<div>
  <div>
    This is Badge component
  </div>
</div>
`;
